### ISSUE TITLE
FR: Officially maintained packages of Tailscale for Asustor NASs

### ISSUE THREAD

**Issue by bowmanjc** [tailscale/tailscale#10106]
### What are you trying to do?

Hardware: [Asustor Lockerstor4 [Gen 1] (AS6604T)](https://www.asustor.com/en/product?p_id=69)
OS: [ADM 4.2.4.RL82](https://www.asustor.com/service/release_notes#ADM%204.2.4.RL82_all)
Concern: [Third-party packaged Tailscale App v1.48.1](https://appdownload.asustor.com/0010_117518_1692947631_tailscale-docker_1.48.1.r01_x86-64.apk)

Like the Synology NAS packages, I'd like to request official packages for Asustor NASs, specifically the Lockerstor Gen 1 & 2 NASs (x86_64), as I don't feel that I can trust a third-party package for this capability. Plus, the third-party package is outdated at version 1.48.1.

### How should we solve this?

Preferably, I'd like for this package to be available directly from Asustor's App Central from within ADM.

### What is the impact of not solving this?

If Asustor end-users install the third-party package thinking that it is an officially supported package and it turns out to be malicious, it could irreparably damage Tailscale's reputation for trustworthiness.

Currently, I'm not able to link my Asustor NAS to my Tailnet.

### Anything else?

Thank you for your time and attention.

**Comment by moringa303** [comment]
Please consider creating an official Tailscale app for Asustor NAS. Their devices are far better than those from Synology and QNAP. Many users would highly appreciate this.
Thanks!

**Comment by lifemelody888** [comment]
I also look forward to an official app for **Asustor NAS**.
Hardware: AS6704T
ADM Version: 4.3.1.R752

I currently use a 3rd-party app in Asustor official app store, named "tailscale 1.68.1 +", shown "Linux 5.13.x" in Tailscale admin console.

**Comment by GreggMendel** [comment]
I too would love to see a genuine Tailscale app for Asustor (AS304T).

**Comment by SmokingGun31** [comment]
Yes, I would also like to get a proper tailscale app for my asustor nas!!

**Comment by moringa303** [comment]
Is there any update on this ?
